src/schema/v2/searchCriteriaLabel.ts

```diff
@@ -0,0 +1,346 @@
+import { GraphQLObjectType, GraphQLString } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { startCase } from "lodash"
+
+import allAttributionClasses from "lib/attributionClasses"
+import { COLORS } from "lib/colors"
+
+type SearchCriteriaLabel = {
+  /** The GraphQL field name of the filter facet */
+  field: string
+
+  /** The human-friendly name of the filter facet */
+  name: string
+
+  /** The human-friendly value of the filter facet */
+  value: string
+}
+
+/**
+ * A type, derived here in Metaphysics from the upstream Gravity response,
+ * that represents a SearchCriteria's filter in a human-friendly label,
+ * suitable for use in a pill UI, for example.
+ */
+export const SearchCriteriaLabel = new GraphQLObjectType<
+  SearchCriteriaLabel,
+  ResolverContext
+>({
+  name: "SearchCriteriaLabel",
+  description:
+    "Human-friendly representation of a single SearchCriteria filter",
+  fields: {
+    field: {
+      type: GraphQLString,
+      description: "The GraphQL field name of the filter facet",
+    },
+    name: {
+      type: GraphQLString,
+      description: "The human-friendly name of the filter facet",
+    },
+    value: {
+      type: GraphQLString,
+      description: "The human-friendly value of the filter facet",
+    },
+  },
+})
+
+/**
+ * A resolver that takes the current SearchCriteria and returns a list of
+ * SearchCriteriaLabels representing that SearchCriteria in human-friendly form
+ */
+export const resolveSearchCriteriaLabels = async (
+  parent,
+  _args,
+  context,
+  _info
+) => {
+  const {
+    artistIDs,
+    attributionClass,
+    additionalGeneIDs,
+    priceRange,
+    sizes,
+    width,
+    height,
+    acquireable,
+    atAuction,
+    inquireableOnly,
+    offerable,
+    materialsTerms,
+    locationCities,
+    majorPeriods,
+    colors,
+    partnerIDs,
+  } = parent
+
+  const { artistLoader, partnerLoader } = context
+
```

Comment:
The artist and partner labels are actually fulfilled by going back to Gravity to request the names belonging to the corresponding IDs that are stored in the SearchCriteria.